src/sentry/eventstream/kafka/dispatch.py

```diff
@@ -45,6 +45,7 @@ def dispatch_post_process_group_task(
     skip_consume: bool = False,
     group_states: GroupStates | None = None,
     occurrence_id: str | None = None,
+    eventstream_type: str | None = None,
```

Comment:
is the nullable temporary?

Reply:
Yes, temporary for now

Reply:
We might need to leave this nullable permanently. When deploying this to self hosted, there could be tasks in the pipeline that would fail after an upgrade if it expected a value here